File: 백준/Bronze/16099. Larger Sport Facility/Larger Sport Facility.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    lt,wt,le,we=map(int,input().split())
    print("Tie" if lt*wt==le*we else "Eurecom" if lt*wt<le*we else "TelecomParisTech")

File: 백준/Bronze/16099. Larger Sport Facility/Larger Sport Facility.cc
#include <iostream>
using namespace std;
int main(void) {
    int n;
    long long lt, wt, le, we;
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> lt >> wt >> le >> we;
        cout << (lt * wt == le * we ? "Tie" : (lt * wt < le * we ? "Eurecom" : "TelecomParisTech")) << "\n";
    }
    return 0;
}